docs/install/install.rst

```diff
@@ -0,0 +1,100 @@
+.. meta::
+  :description: Install ROCm SMI
+  :keywords: install, SMI, library, api, AMD, ROCm
+
+
+*********************
+Installing ROCm SMI
+*********************
+
+Planned Deprication Notice 
+ROCm System Management Interface (ROCm SMI) Library is planned to be ***depricated***. Release date to be announced soon. Please start migrating to AMD SMI.
+  - Documentation: [https://rocm.docs.amd.com](https://rocm.docs.amd.com/projects/amdsmi/en/latest/)
```

Comment:
The doc build failure flags this line:
```
/home/docs/checkouts/readthedocs.org/user_builds/advanced-micro-devices-rocm-smi-lib/checkouts/168/docs/install/install.rst:12: ERROR: Unexpected indentation.
/home/docs/checkouts/readthedocs.org/user_builds/advanced-micro-devices-rocm-smi-lib/checkouts/168/docs/install/install.rst:40: ERROR: Content block expected for the "Note" directive; none found.
```

This is Markdown syntax

Reply:
RST syntax for links: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

Reply:
Thanks, @samjwu. Incorrect Markdown syntax has not caused a build error or failure before. It just displays incorrectly in the file. This is good to know.